lldb/bindings/interface/SBProgressDocstrings.i

```diff
@@ -12,3 +12,10 @@ and will always send an initial progress update, updates when
 Progress::Increment() is called, and also will make sure that a progress
 completed update is reported even if the user doesn't explicitly cause one
 to be sent.") lldb::SBProgress;
+
+%feature("docstring",
```

Comment:
@JDevlieghere would you mind providing your wordsmithing expertise?

Reply:
How about something like:

> Finalize SBProgress, which will cause a progress end event to be emitted. This happens automatically when the SBProcess object is destroyed, but can be done explicitly with Finalize to avoid having to rely on the language semantics for destruction.